.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,15 +10,15 @@ assignees: ""
 A clear and concise description of what the bug is.
 If you're planning on posting a screenshot, please paste the accompanying code here.
 
-**Screenshot**
+**Screenshot**   
```

Comment:
Where are these spaces coming from? Do you have prettier installed? Do I need to add some rules?

Reply:
I assume prettier changed it. Not sure what rule it is though